Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-http.tsx

```diff
@@ -10,6 +10,7 @@ interface IErrorDataType {
     title: string;
     status: number;
     detail: string;
+    extensions: IDictionary<string>;
```

Comment:
What if we want to return numbers or object? 
Note: this is nice info https://stackoverflow.com/a/67290755

Reply:
Changed to IDictionary<object>.